src/html2ans/parsers/utils.py

```diff
@@ -48,10 +48,7 @@ class AbstractParserUtilities(object):
     List of tags considered empty (if a tag passed to ``is_empty`` is in this list,
     ``is_empty`` will return True).
     """
-
-    TEXT_TAGS = [
-        'blockquote',
-        'p',
+    STYLING_TAGS = [
```

Comment:
Can we call this something else? `a` is not a styling tag; this is more like "inline tags we want to leave inline"